avaframe/com1DFAPy/timeDiscretizations.py

```diff
@@ -0,0 +1,50 @@
+"""
+    function regarding time discretization and time stepping for com1DFA
+
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import glob
+import logging
+import numpy as np
+
+# Local imports
+import avaframe.in3Utils.fileHandlerUtils as fU
+import avaframe.in2Trans.ascUtils as IOf
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def getTimeStep(particles, dem, cfg):
```

Comment:
Maybe make it more desciptive by naming int getTimeStepCFL or something similar?

Reply:
done